examples/azure/terraform-azure-example/main.tf

```diff
@@ -1,14 +1,16 @@
 # ---------------------------------------------------------------------------------------------------------------------
-# DEPLOY AN AZURE VIRTUAL MACHINE
-# This is a basic example of how to deploy an Azure Virtual Machine with the minimum network resources.
+# DEPLOY A SIMPLE AZURE VIRTUAL MACHINE
+# This is a simple example of how to deploy an Azure Virtual Machine with the minimum network resources.
```

Comment:
NIT: We've been trying to avoid using words like `simple`, `just`, or `easy` in docs as it doesn't really add anything, and is not very newbie friendly. The main reason is that it can make users feel bad if they actually don't understand what is going on: "oh if this is the simple example, and I don't understand what is going on, there is no way I can learn this..."

See [this talk](https://www.writethedocs.org/videos/prague/2018/don-t-say-simply-jim-fisher/) for more info on this.

I think this is perfectly fine as `DEPLOY AN AZURE VIRTUAL MACHINE`. In fact, I would probably drop `basic` from the original text too.

Reply:
Updated, makes sense. Thanks for the link :)